native/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -205,6 +220,21 @@ private void generateConfigurationBalFile() throws BalException {
         }
     }
 
+    private void copyEntities(ArrayList<Entity> entityArray, ArrayList<ModuleMemberDeclarationNode> moduleMembers,
+                              ArrayList<ImportDeclarationNode> importArray)
+            throws BalException {
+        SyntaxTree copiedEntitiesTree = BalSyntaxTreeGenerator.copyEntities(moduleMembers, importArray);
+        try {
+            writeOutputFile(copiedEntitiesTree, Paths.get(this.sourcePath, "modules",
+                            "clients", "entities.bal")
```

Comment:
same

Reply:
updated in [a7041fd](https://github.com/ballerina-platform/persist-tools/pull/31/commits/a7041fdbccf9572781cd185694e8cb15352d4e0b)